rebar.config

```diff
@@ -1,5 +1,12 @@
+{cover_export_enabled, true}.
+
+{cover_enabled, true}.
+
 {deps, [
-    {lager, "3.9.2"}
+    {grpcbox, {git, "https://github.com/novalabsxyz/grpcbox.git"}},
+    {helium_proto, {git, "https://github.com/helium/proto.git"}},
+    {lager, "3.9.2"},
+    {lorawan, {git, "https://github.com/helium/erlang-lorawan.git"}}
```

Comment:
I don't think we need this one in there I ll be adding it as part of another PR

Reply:
Removed dependency.  Fixed in commit 3884fc8